packages/patternfly-4/react-core/src/components/ContextSelector/ContextSelectorMenuList.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/ContextSelector/context-selector.css';
```

Comment:
Change to `@patternfly/react-styles/components/ContextSelector/context-selector`